app/assets/i18n/strings_pt-BR.json

```diff
@@ -28,7 +28,7 @@
     "open": "Abrir",
     "queue": "Fila",
     "quickSave": "Salvamento Rápido",
-    "quickSaveFromFavorites": "Salvamento rápido para \"Favoritos\"",
+    "quickSaveFromFavorites": "salvar rapidamente dos \"Favoritos\"",
```

Comment:
Should this start with capital `S` also?

Reply:
Technically, in portuguese, every phrase/sentence needs to starts with a capital letter.
In that way, yes, it should be a capital `S`.